README.md

```diff
@@ -23,18 +23,23 @@ Or clone the repo:
     $ git clone https://github.com/uvjustin/pyalarmdotcomajax.git
     $ python setup.py install
 
+Note that you'll still need `aiohttp` and `bs4` to be installed somehow (e.g. `pip3 install aiohttp`)
+
 Sensor deviceTypes
 ------------------
 
 This list identifies deviceTypes used in the alarm.com API and is incomplete. Please help by submitting missing values.
 
-| deviceType | Description          |
-|------------|----------------------|
-| 1          | Contact Sensor       |
-| 5          | Smoke Detector       |
-| 6          | CO Detector          |
-| 9          | Panic Button         |
-| 19         | Glass Break Detector |
+| deviceType | Description                 |
+|------------|-----------------------------|
+| 1          | Contact Sensor              |
+| 2          | Motion Sensor               |
+| 5          | Smoke Detector              |
+| 6          | CO Detector                 |
+| 9          | Panic Button                |
+| 19         | Glass Break Detector        |
+| 68         | Panel(?) Image Sensor       | 

```

Comment:
Not sure on this one. On my system, panel motion detector != motion sensor, so I _bet_ it's different but not 100% positive.